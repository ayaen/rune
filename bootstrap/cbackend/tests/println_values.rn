//  Copyright 2024 Google LLC.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
s = "hello"

println s
println -1i8
println 65535u16
println 0xdeadbeefu32
println 1
println -1

println 2.71828175f32
println -3.1415926535897931

println "this ", "is a bool:", true, "|", false, " ", -1, ", ", 3.141

print "boo: "
println 3
