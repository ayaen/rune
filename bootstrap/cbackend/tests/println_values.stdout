hello
-1
65535
3735928559
1
-1
2.7182817500000001
-3.1415926535897931
this is a bool:true|false -1, 3.141
boo: 3
